### DISCUSSION TITLE
重绑定保护，白名单希望支持泛域名

### DISCUSSION THREAD

**Discussion by SyberRabbit** [istoreos/istoreos#2024] [Ideas]
如这个帖子，
https://github.com/istoreos/istoreos/discussions/2023

**Comment by jjm2473** [comment]
直接关掉“重绑定保护”就行了。能否支持泛域名应该要查dnsmasq文档，不支持也要找他们反馈。

**Comment by jjm2473** [comment]
如果你的意思是包含子域名，那么本来就是如此，不需要任何修改。
